src/it/java/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/RightCurlyTest.java

```diff
@@ -37,6 +36,11 @@ protected String getPackageLocation() {
         return "com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks";
     }
 
+    private Configuration getModuleConfigs() {
```

Comment:
`CheckstyleTestMakeupCheck` complains about this, looking for a decision how we should handle this. `CheckstyleTestMakeupCheck` was to ensure tests follow a pattern so we can pull the information from configs and such for automated configuration creation. I created this method to ease loading so many IDs for this class and to ensure 1 was not missed.

Reply:
@romani ping

Reply:
in worse scenario - ok to suppress.

Reply:
Done.